docs/2_2_common_mechanisms.adoc

```diff
@@ -834,6 +834,10 @@ For restrictions on when to call which of the following functions for <<table-ov
 
 <<fmi3SetClock>> must not be called in Scheduled Execution, instead <<fmi3ActivateModelPartition>> must be called.
 
+The Clocks API function argument names <<intervals>>, <<shifts>>, <<counters>> and <<resolutions>> are chosen to match the link:https://specification.modelica.org/maint/3.6/synchronous-language-elements.html#clock-constructors[Modelica Clock Constructors].
+However, the FMI standard does not assume that the importer tool's clock implementation matches the Modelica Clock implementation.
+In general, the arguments <<counters>> and <<resolutions>> have the meaning of a "numerator" and a "denominator".
```

Comment:
```suggestion
_[The Clocks API function argument names <<intervals>>, <<shifts>>, <<counters>> and <<resolutions>> are chosen to match the link:https://specification.modelica.org/maint/3.6/synchronous-language-elements.html#clock-constructors[Modelica Clock Constructors]._
_However, the FMI standard does not assume that the importer tool's clock implementation matches the Modelica Clock implementation._
_In general, the arguments <<counters>> and <<resolutions>> have the meaning of a "numerator" and a "denominator".]_
```
The explanation should be non-normative.

Reply:
```suggestion
_[The Clocks API function argument names <<intervals>>, <<shifts>>, <<counters>> and <<resolutions>> are chosen to match the naming used in link:https://specification.modelica.org/maint/3.6/synchronous-language-elements.html#clock-constructors[Modelica Clock Constructors]._
_This commonality of naming does not imply any normative reference to the semantics of Modelica._
_Mathematically speaking, the arguments <<counters>> and <<resolutions>> in the fraction variant of these functions serve as numerators and denominators of fractional representations of the corresponding floating-point number arguments of the decimal variant.]_
```

Maybe the final sentence should be split up a bit more, but didn't find the fortune to do so.